workflows/laava/inputs.md

```diff
@@ -26,10 +27,12 @@ For a folder of multiple samples (post-demux):
 The remaining parameters are shared by all samples:
 
 - Vector annotation (BED)
-  - Annotated vector construct region coordinates in 4-column BED format. Must include 1
-    region with the label 'vector', indicating the payload region that spans ITR to ITR.
-    May also include additional labeled regions, e.g.  for ITR, promoter, and CDS
-    regions, but these will not affect the output.
+  - Annotated vector construct region coordinates in 4-column UCSC BED format.
+  - This file must indicate the transgene/payload region via either two labeled Inverted
+    Terminal Repeat (ITR) regions (see itr_label_1 and itr_label_2 below) or, as a
+    legacy mode, one region with the label 'vector', spanning both ITRs (inclusive).
+  - May also include additional labeled regions, e.g. for promoter and CDS regions;
+    these will be ignored and will not affect the output.
```

Comment:
We should think about adding a minimal BED file information, in the documentation.

Reply:
What additional information would you like to see here? An explanation of the BED format?